docs/change_node_hostname.rst

```diff
@@ -0,0 +1,103 @@
+Changing the hostname of a MetalK8s node
+========================================
+
+
+#. On the node, change the hostname:
+
+   .. code-block:: shell
+
+      $ hostnamectl set-hostname <New hostname>
+      $ systemctl restart systemd-hostnamed
+
+#. Check that the hostname has changed
+
+   .. code-block:: shell
+
+      $ hostnamectl status
+
+      Static hostname: controlplane2
+      Pretty hostname: ControlPlane2
+        Icon name: computer-vm
+         Chassis: vm
+      Machine ID: 5003025f93c1a84914ea5ae66519c100
+         Boot ID: f28d5c64f06c48a3a775e24c4f03d00c
+         Virtualization: kvm
+      Oerating System: CentOS Linux 7 (Core)
+         CPE OS Name: cpe:/o:centos:centos:7
+             Kernel: Linux 3.10.0-957.12.2.el7.x86_64
+         Architecture: x86-64
+
+
+#. On the bootstrap node, verify the hostname change allowed a change of status on the bootstrap,
```

Comment:
`#. On the bootstrap node, check the hostname edition allowed a change of status on the bootstrap. The node must be in a **NotReady** status.`